parquet-format-structures/src/main/java/org/apache/parquet/format/Util.java

```diff
@@ -55,36 +59,88 @@
 public class Util {
 
   public static void writeColumnIndex(ColumnIndex columnIndex, OutputStream to) throws IOException {
-    write(columnIndex, to);
+    writeColumnIndex(columnIndex, to, (BlockCipher.Encryptor) null, (byte[]) null);
```

Comment:
I think you don't need to enforce casting here. It will be done by implicit casting.